plugins/funds.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+""" This plugin gives you a nicer overview of the funds that you own.
+
+Instead of calling listfunds and adding all outputs and channels
+this plugin does that for you. 
+
+Author: Rene Pickhardt (https://ln.rene-pickhardt.de)
+
+"""
+
+import json
+import sys
+
+from lightning.lightning import LightningRpc
+from os.path import expanduser
+
+path = expanduser("~/.lightning/lightning-rpc")
```

Comment:
I was talking explicitly about this line. My feeling is that I should be able to send RPC calls via json directly to the lightningdaemon. However I don't know the format of the json and it is not documented yet.

Reply:
#2131 will contain a `configuration` field in the `configure` call that'll explicitly tell you were to connect :wink:

Reply:
#2131 didn't?  I have a patch which adds a 'rpc_filename' to the init msg however, will push (I needed this too, of course)!

Reply:
So I will be able to find this in def json_init() in the options argument?

How can I look at these variables? Since stdout is sent to lightningd I can't print variables to inspect them which is annoying

Reply:
Wait, where did that patch end up?